packages/cf-grid/src/cf-grid.less

```diff
@@ -12,10 +12,10 @@
 // Less variables
 //
 
-@grid_wrapper-width:            1230px;
-@grid_gutter-width:             30px;
-@grid_total-columns:            12;
-@grid_debug:                    false;
+@grid_wrapper-width: 1230px;
```

Comment:
Matches formatting in https://github.com/cfpb/capital-framework/blob/master/packages/cf-grid/src-generated/cf-grid-generated.less#L27